### PR TITLE
Group task

### PR DESCRIPTION
**Pull request by sentayhu19**
**In this PR**

- [x] Create Item class in a separate .rb file.
- [x] All Item class properties visible in the diagram should be defined and set up in the constructor method. Exception: 
- [x] properties for the 1-to-many relationships should NOT be set in the constructor method. Instead, they should have a - 
- [x]  custom setter method created.
- [x] Add all methods visible in the diagram.
- [x] Implement methods:
- [ ] can_be_archived?() in the Item class
should return true if published_date is older than 10 years
otherwise, it should return false
- [ ] move_to_archive() in the Item class
should reuse can_be_archived?() method
should change the archived property to true if the result of the can_be_archived?() method is true
should do nothing if the result of the can_be_archived?() method is false

### CODE REVIEW

File: item.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class Item
+    attr_accessor :genre, :author, :source, :label, :publish_date, :archived
+    def initialize(genre, author, source, label, publish_date, archived)
+        super(genre, author, source, label, publish_date, archived)
```
Comment (LionRouge1) [inline item.rb:4]: I think we don't need the `super` keyword for now what did you think?

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @sentayhu19 and @moise-mulungu,
Good job so far :+1: 
But I can see that you forget about the linter :smile: 
- Please kindly fix linter errors it makes your code unclean.

**Review by LionRouge1 — APPROVED** [review]
Hi @sentayhu19 and @moise-mulungu ,

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! tada

Cheers and Happy coding!clapclapclap

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Review by moise-mulungu — APPROVED** [review]
Good job by implementing all the required tasks :+1: 
- No linter :heavy_check_mark: 
- All methods are ok :heavy_check_mark: 
- All options are well structured :heavy_check_mark:

**Pull request merged**